docs/source/API/core/view/view_alloc.rst

```diff
@@ -44,8 +46,27 @@ Description
 
    ``args`` : Can only be a pointer to memory.
 
+
 .. cppkokkos:type:: ALLOC_PROP
 
    :cppkokkos:type:`ALLOC_PROP` is a special, unspellable implementation-defined type that is returned by :cppkokkos:func:`view_alloc`
    and :cppkokkos:func:`view_wrap`. It represents a bundle of allocator parameters, including the View label, the memory space instance,
    the execution space instance, whether to initialize the memory, whether to allow padding, and the raw pointer value (for wrapped unmanaged views).
+
+.. cppkokkos:type:: WithoutInitializing
+
+   :cppkokkos:type:`WithoutInitializing` is intended to be used in situation where default construction of `View` elements in its
+   associated execution space is not viable. This includes situations such as the construction of objects with virtual functions,
+   or `Views` of elements which do not have a default constructor. Typically, this option is used in conjunction with manual in-place `new`
+   construction of objects and manual destruction of elements.
+
+.. cppkokkos:type:: SequentialHostInit
```

Comment:
Is there any restriction about the memory space of the view ?

Reply:
Yeah there is right now needs to be host accessible.